tgui/packages/tgui/interfaces/AresInterface.js

```diff
@@ -0,0 +1,1499 @@
+import { useBackend } from '../backend';
+import { Flex, Box, Section, Button, Stack } from '../components';
+import { Window } from '../layouts';
+
+const PAGES = {
+  'login': () => Login,
+  'main': () => MainMenu,
+  'announcements': () => AnnouncementLogs,
+  'bioscans': () => BioscanLogs,
+  'bombardments': () => BombardmentLogs,
+  'apollo': () => ApolloLog,
+  'access_log': () => AccessLogs,
+  'delete_log': () => DeletionLogs,
+  'talking': () => ARESTalk,
+  'deleted_talks': () => DeletedTalks,
+  'read_deleted': () => ReadingTalks,
+  'security': () => Security,
+  'requisitions': () => Requisitions,
+  'antiair': () => AntiAir,
+  'emergency': () => Emergency,
+};
+
+export const AresInterface = (props, context) => {
+  const { data } = useBackend(context);
+  const { current_menu } = data;
+  const PageComponent = PAGES[current_menu]();
+
+  return (
+    <Window theme="crtblue" width={780} height={725}>
+      <Window.Content scrollable>
+        <PageComponent />
+      </Window.Content>
+    </Window>
+  );
+};
+
+const Login = (props, context) => {
+  const { act } = useBackend(context);
+
+  return (
+    <Flex
+      direction="column"
+      justify="center"
+      align="center"
+      height="100%"
+      color="darkgrey"
+      fontSize="2rem"
+      mt="-3rem"
+      bold>
+      <Box fontFamily="monospace">ARES v3.2 Interface</Box>
+      <Box mb="2rem" fontFamily="monospace">
+        WY-DOS Executive
+      </Box>
+      <Box fontFamily="monospace">Version 8.2.3</Box>
+      <Box fontFamily="monospace">Copyright © 2182, Weyland Yutani Corp.</Box>
+
+      <Button
+        content="Login"
+        icon="id-card"
+        width="30vw"
+        textAlign="center"
+        fontSize="1.5rem"
+        p="1rem"
+        mt="5rem"
+        onClick={() => act('login')}
+      />
+    </Flex>
+  );
+};
+
+const MainMenu = (props, context) => {
+  const { data, act } = useBackend(context);
+  const {
+    logged_in,
+    access_text,
+    last_page,
+    current_menu,
+    access_level,
+    sudo,
+  } = data;
+
+  return (
+    <>
+      <Section>
```

Comment:
you should probably split these separate segments into different components to avoid the gigantic component and the top level fragment, this applies for all your components